server/views/pages/overview/partials/goalsTab/goalsTabContents.njk

```diff
@@ -164,24 +164,25 @@
       {% endif %}
     </div>
 
-    <div class="govuk-grid-column-one-third app-u-print-full-width">
-      {{ actionsCard({
-        actions: [
-          { 
-            title: '<span class="goal-action"><img src="/assets/images/icon-goal.svg" role="presentation" alt="" width="35px" height="28px" class="goal-icon" /> Add a new goal</span>' | safe, 
-            href: '/plan/' ~ prisonerSummary.prisonNumber ~ '/goals/create', 
-            id: 'add-goal-button', 
-            'render-if': hasEditAuthority,
-            attributes: {
-              'data-qa': 'add-goal-button'
+    {% if hasEditAuthority %}
```

Comment:
I'm not sure about hiding the actions card like this 🤔 

Why are we doing it? (in context of the story/PR that we parked because we decided that we needed to fix the problem with Change/Add links on the Education & Training and Work Interest tabs)
I accept the fact that a read-only user will see an empty actions card, but lets fix that separately. I have a funny feeling that there are some actions that we will need to add in future stories that are relevant and appropriate for read-only users, so you're proposed change here will cause us a problem there.